model/job_test.go

```diff
@@ -222,3 +223,36 @@ func TestJobsSort(t *testing.T) {
 	assert.Equal(jobs[0].Name, "ccc")
 	assert.Equal(jobs[1].Name, "ddd")
 }
+
+func TestJobsProperties(t *testing.T) {
+	assert := assert.New(t)
+
+	workDir, err := os.Getwd()
+	assert.Nil(err)
```

Comment:
Prefer `assert.NoError` over `assert.Nil` when testing for errors (they give better messages on failure).